lib/index.js

```diff
@@ -25,11 +25,19 @@ class EvervaultClient {
     PRIME256V1: 'prime256v1',
   };
 
-  constructor(apiKey, options = {}) {
-    if (!Datatypes.isString(apiKey)) {
-      throw new errors.InitializationError('API key must be a string');
+  constructor(appId, apiKey, options = {}) {
```

Comment:
In the type file `evervault.d.ts`, the `appId` param is `appUuid`

Reply:
updated